packages/skeleton-react/src/lib/components/Tabs/Tabs.test.tsx

```diff
@@ -1,234 +1,151 @@
-// import { describe, expect, it, vi } from 'vitest';
-// import { act, render } from '@testing-library/react';
-// import { Tabs } from './Tabs.js';
-// import userEvent from '@testing-library/user-event';
-
 import { describe, expect, it } from 'vitest';
 import { render } from '@testing-library/react';
-import { Tabs } from './Tabs.js';
-
-// // *************************
-// // Integration Tests
-// // *************************
-
-// describe('Tabs usage', () => {
-// 	it('should show the panel when the control is clicked', async () => {
-// 		const setGroup = vi.fn();
-
-// 		const tabsComponent = (group: string) => (
-// 			<Tabs>
-// 				<Tabs.List>
-// 					<Tabs.Control name="test1" group={group} onChange={setGroup}>
-// 						test control 1
-// 					</Tabs.Control>
-// 					<Tabs.Control name="test2" group={group} onChange={setGroup}>
-// 						test control 2
-// 					</Tabs.Control>
-// 				</Tabs.List>
-// 				<Tabs.Panel group={group} value="test1">
-// 					test Panel 1
-// 				</Tabs.Panel>
-// 				<Tabs.Panel group={group} value="test2">
-// 					test Panel 2
-// 				</Tabs.Panel>
-// 			</Tabs>
-// 		);
-
-// 		const { getByText, queryByText, rerender } = render(tabsComponent('test1'));
-
-// 		const control1 = getByText('test control 1');
-// 		const control2 = getByText('test control 2');
-
-// 		// expect controls to be visible
-// 		expect(control1).toBeInTheDocument();
-// 		expect(control2).toBeInTheDocument();
-
-// 		// expect first panel to be visible
-// 		expect(queryByText('test Panel 1')).toBeInTheDocument();
-
-// 		// expect second panel to be hidden
-// 		expect(queryByText('test control 2')).toBeInTheDocument();
 
-// 		// click second control
-// 		await act(async () => {
-// 			await userEvent.click(control2);
-// 		});
-
-// 		// make sure setGroup is called after click
-// 		expect(setGroup).toHaveBeenCalledWith('test2');
-// 		// rerender the component with the new group value
-// 		rerender(tabsComponent('test2'));
-
-// 		// expect first panel to be hidden
-// 		expect(queryByText('test Panel 1')).not.toBeInTheDocument();
-// 		expect(queryByText('test Panel 2')).toBeInTheDocument();
-// 	});
-// });
-
-// // *************************
-// // Unit Tests
-// // *************************
+import { Tabs } from './Tabs.js';
 
-// // Tabs ---
+// Tabs (integration)
 
 describe('<Tabs>', () => {
-	it('should render the component', () => {
-		const { getByTestId } = render(<Tabs />);
-		expect(getByTestId('tabs')).toBeInTheDocument();
+	it('should render the entire component group', () => {
```

Comment:
I think one test for this might be useful to ensure things like context is being pass through the component tree. But I don't think they're a hard requirement.

Reply:
That's true, let's keep them.